src/storage/ring_buffer.rs

```diff
@@ -72,6 +72,11 @@ impl<'a, T: 'a> RingBuffer<'a, T> {
         self.capacity() - self.len()
     }
 
+    /// Return the number of contiguos elements that can be added to the ring buffer.
```

Comment:
This comment doesn't make sense.

    /// Return the largest number of elements that can be added to the buffer
    /// without wrapping around (i.e. in a single `enqueue_many` call).